mappings/net/minecraft/world/gen/stateprovider/DualNoise2DBlockStateProvider.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_6578 net/minecraft/world/gen/stateprovider/DualNoise2DBlockStateProvider
```

Comment:
I recommend changing `2D` to `2d` in the class name

Reply:
True, that would be more readable. I think I'll just leave out the "2D", because they're actually 3D (which is probably unintended, but it's still confusing if the class is named 2D in that case).